protocol/x/clob/types/types.go

```diff
@@ -1 +1,63 @@
 package types
+
+import "math/big"
+
+type FillForProcess interface {
+	TakerAddr() string
+	TakerFeeQuoteQuantums() *big.Int
+	MakerAddr() string
+	MakerFeeQuoteQuantums() *big.Int
+	FillQuoteQuantums() *big.Int
+	PerpetualId() uint32
```

Comment:
The original intention of having this interface is so it applies to both perpetual and spot in the future. If we calling direct calling this `PerpetualId` at interface level, might as we just delete `FillForProcess` and use `PerpetualFillForProcess` struct directly